az-core/src/main/java/azkaban/utils/Props.java

```diff
@@ -430,6 +430,13 @@ public String getString(final String key) {
   }
 
   /**
+   * Returns a list of clusters with the comma as the separator of the value
+   */
```

Comment:
Provide an example to clarify whats parsed by the method.

Reply:
Add unit tests to cover different cases.

Reply:
+1 on Unit Test! You might need to create a test file under az-core module, as we don't have one.